data/locations.yml

```diff
@@ -0,0 +1,10 @@
+---
+"25":
```

Comment:
Can we make this name a bit more explicit? Seeing just `25` in the templates is rather unclear unless you're already familiar with the venues. Perhaps `building-25`?